pages/classes/mighty.md

```diff
@@ -81,10 +83,53 @@ You can use your [Reaction](../../pages/combat/reactions.md) to impose disadvant
 
 <header>
 
-### Surge: Defend
+### Surge: Bolster
 
 <p class="subheading">1st-level Defender archetype feature</p>
 
 </header>
 
-When a **Close** ally is hit by a melee attack, you can use your [Reaction](../../pages/combat/reactions.md) to reduce the damage by **1d10 + your level**.
+As a [Bonus Action](pages/combat/bonus-actions.md) you can restore [Stamina](pages/combat/stamina.md) equal to **1d10 + your level** to a **Close** ally.
```

Comment:
Required some re-working on the Defender's Block surge. Now the surge is healing (more in-line with Second Wind), which allows us to re-use block mechanic as a "Tactic" below.